oneview/data_source_hypervisor_cluster_profile.go

```diff
@@ -0,0 +1,528 @@
+															"subnet_mask": {
+																Type:     schema.TypeString,
+																Computed: true},
+															"virtual_port_purpose": {
+																Type:     schema.TypeList,
+																Computed: true,
+																Elem: &schema.Schema{
+																	Type: schema.TypeString},
+															},
+														},
+													}},
+												"vlan": {
+													Type:     schema.TypeString,
+													Computed: true},
+											}},
+									},
+									"virtual_switch_type": {
+										Type:     schema.TypeString,
+										Computed: true},
+									"virtual_switch_uplinks": {
+										Type:     schema.TypeList,
+										Computed: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"action": {
+													Type:     schema.TypeString,
+													Computed: true},
+												"active": {
+													Type:     schema.TypeBool,
+													Computed: true},
+												"mac": {
+													Type:     schema.TypeString,
+													Computed: true},
+												"name": {
+													Type:     schema.TypeString,
+													Computed: true},
+												"vmnic": {
+													Type:     schema.TypeString,
+													Computed: true},
+											},
+										},
+									},
+								},
+							}},
+					}}},
+			"hypervisor_cluster_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"hypervisor_host_profile_uris": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"hypervisor_manager_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"hypervisor_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"ip_pools": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString},
+				Set: schema.HashString,
+			},
+
+			"mgmt_ip_settings_override": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"modified": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"path": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"refresh_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"scopes_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"shared_storage_volumes": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString},
+				Set: schema.HashString,
+			},
+
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"state_reason": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"uri": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Document(docs/d) says that uri is mandatory, but in this file name is mentioned as mandatory and uri as optional. Please fix this.

Reply:
Fixed.